internal/utils.go

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
```

Comment:
Let's not have any generic "utils" or similar file. Create well named appropriate packages to house these methods. And let's move the docker code out of internal package as well. It should be its own package. Though try to come up with a name that doesn't conflict/confuse with the github.com/docker package

Reply:
Moved NanosecondsToSec and BytesToMB to math package and RelativeToAbsolutePath to filepath package